packages/components/src/select/src/Select.css

```diff
@@ -6,43 +6,49 @@
     cursor: pointer;
     appearance: none;
     border: none;
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
     transition: all var(--o-ui-easing-duration-2) var(--o-ui-easing-productive);
     display: inline-flex;
     justify-content: flex-end;
     align-items: center;
-    min-height: var(--o-ui-sz-6);
+    min-height: 2.5rem;
     overflow: hidden;
-    padding: 0 var(--o-ui-sp-3) 0;
-    border-radius: var(--o-ui-br-2);
-    line-height: inherit;
-    font-family: inherit;
-    font-size: 100%;
+    padding: 0 var(--hop-space-inset-md);
+    border-radius: var(--hop-shape-rounded-md);
+    line-height: var(--hop-body-sm-line-height);
+    font-family: var(--hop-body-sm-font-family);
+    font-size: var(--hop-body-sm-font-size);
 }
 
 /* View https://css-tricks.com/slightly-careful-sub-elements-clickable-things */
 .o-ui-select-trigger > * {
     pointer-events: none;
 }
 
+/* TRIGGER | ICONS */
+.o-ui-select-trigger .o-ui-icon {
+    height: 1rem;
+    width: 1rem;
```

Comment:
You could remove height and swap is for aspect-ratio: 1/1; I know that Franck always does this.

Reply:
I suppose the support is pretty good now, but I think I'll keep as is for now.